### PR TITLE
Attach polarion Id(s) to RHCeph 5x cephadm test cases

### PR DESCRIPTION
**Pull request by sunilkumarn417**
Signed-off-by: sunilkumarn417 <sunnagar@redhat.com>

- Attached polarion Id(s) to cephadm test cases

### CODE REVIEW

**Comment by veera-raghava-reddy** [comment]
@sunilkumarn417 There are multiple PolarionIDs mapped to one automated test. Are they the same tests, if Yes should we remove duplicate tests in Polarion and have 1-to-1 mapping from Polarion to Autoamtion scripts?
If different tests, how to differentiate which specific test has failed?

**Comment by sunilkumarn417** [comment]
> @sunilkumarn417 There are multiple PolarionIDs mapped to one automated test. Are they the same tests, if Yes should we remove duplicate tests in Polarion and have 1-to-1 mapping from Polarion to Autoamtion scripts? If different tests, how to differentiate which specific test has failed?

Yes @veera-raghava-reddy, I agree on your point. Currently our bootstrap test case covers multiple sub-options and along with the validations, so had attached multiple test case Ids for some of the test cases. This has reduced number of test suites under respective tier-level and aligned with our pipeline time constraint.
I will re-look this test cases again and will try to map 1:1 with test cases we have automated.

**Comment by Ameenasuhani** [comment]
@sunilkumarn417  for `Add new host node with IP address`  and `Add new host node with IP address and labels` do you think we can create a new test case and attach it instead of this in some places?

Verify list of hosts added to the cluster with [plain\|json][json-pretty\|yaml]
https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573776 :

File: suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml [hunk 1]
```diff
@@ -218,7 +218,7 @@ tests:
   - test:
       name: Cephadm-ansible purge playbook
       desc: purge cephadm cluster using cephadm-purge-cluster.yml playbook
-      polarion-id: CEPH-83574414
+      polarion-id: CEPH-83574414,CEPH-83573765
```
[the remark this answers is not in the document]
Comment (sunilkumarn417) [reply suites/pacific/upgrades/tier-1_upgrade_cephadm.yaml:221]: purge-cluster playbook uses same command `cephadm rm-cluster` to purge.

**Comment by sunilkumarn417** [comment]
> @sunilkumarn417 for `Add new host node with IP address` and `Add new host node with IP address and labels` do you think we can create a new test case and attach it instead of this in some places?
> 
> Verify list of hosts added to the cluster with [plain|json][json-pretty|yaml] https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573776 :

Yeah, for few them we could add new test cases. But as per this PR, trying to attach polarion Ids without changing test cases in suites.

**Review by Ameenasuhani — APPROVED** [review]

**Comment by Ameenasuhani** [comment]
LGTM

**Comment by sunilangadi2** [comment]
LGTM.

**Review by sunilangadi2 — APPROVED** [review]

**Review by psathyan — APPROVED** [review]

**Pull request merged**